src/main/pages/portable-workspaces/factories-getting-started.adoc

```diff
@@ -8,62 +8,69 @@ folder: portable-workspaces
 ---
 
 
-A Factory is a template containing configuration to automate generating new Workspace with factory identifier added to IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
+A factory is a template containing configuration to automate generating new Workspace with factory identifier added to the IDE URL. Factories can be used to create replicas of existing workspaces or to automate the provisioning of statically or dynamically defined workspaces.
 
 [id="try-a-factory"]
-== Try a Factory
+== Trying a factory
 
-Clone a public workspace on `che.openshift.io` by clicking the button below. https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[image:https://che.openshift.io/factory/resources/factory-contribute.svg[Try a Factory]]
+To clone a public workspace on `che.openshift.io`, click the button: https://che.openshift.io/f?id=factorymtyoro1y0qt8tq2j[image:https://che.openshift.io/factory/resources/factory-contribute.svg[Try a Factory]].
 
 [id="using-factories"]
-== Using Factories
+== Using factories
 
-Factories can be invoked from a Factory URL built in multiple ways. You can replace the {% if site.product_mini_cli=="codenvy" %}`che.openshift.io` domain with the hostname of any Codenvy on-premises installation.{% else %}`localhost:8080` domain with the hostname of any Che installation.{% endif %}
+Factories can be invoked from a factory URL built in multiple ways. You can replace the {% if site.product_mini_cli=="codenvy" %}`che.openshift.io` domain with the hostname of any Codenvy on-premises installation.{% else %}`localhost:8080` domain with the hostname of any Che installation.{% endif %}
 
-{% if site.product_mini_cli=="codenvy" %}Using Factories on `che.openshift.io` require the user to be authenticated. Users who are not authenticated will be presented a login screen after they click on the Factory URL. Users without an account can create one using the same dialog.{% else %}{% endif %}
+{% if site.product_mini_cli=="codenvy" %}To use factories on `che.openshift.io`, you must be an authenticated user. Users who are not authenticated will be presented with a login screen after they click on the factory URL. Users without an account can create one using the same dialog.{% else %}{% endif %}.
 
 [id="invoke-factory-using-its-unique-hashcode"]
-== Invoke Factory using its unique hashcode
+== Invoking a factory using its unique hashcode
+
+To invoke a factory, use the following format:
+----
+/f?id={hashcode} + 
+/factory?id={hashcode}
+----
```

Comment:
Is there a reason we removed the tables everywhere, the earlier version looks neat.